invenio_app_ils/ui/src/invenio_app_ils/common/config.js

```diff
@@ -8,4 +8,4 @@ export const invenioConfig = getInvenioConfig();
 
 // NOTE: the delay that is used when we fire a request with setTimeOut
 // in order to give some time to elastic search to index.
-export const TIMEOUT_DELAY = 3000;
+export const ES_DELAY = 3000;
```

Comment:
Thanks!
Just an idea: why not exposing the python config var `ILS_INDEXER_TASK_DELAY`? It is doing the same thing for the backend part. Or better to keep them separated?

Reply:
It would make sense to merge them but `ILS_INDEXER_TASK_DELAY` is currently set to 5 seconds while this is 3 seconds. 5 seconds might feel quite long in the UI.

Reply:
I believe that the value of `5` for the `ILS_INDEXER_TASK_DELAY` is an arbitrary choice, we can try to merge them and drop the default value to `3`. @ntarocco I somehow missed that config variable.

Reply:
You are right that 5 secs is not acceptable for the UI, we should anyway decrease it to 2/3 secs. I am also not sure if it makes sense to merge, but in any case I would set this in a python config so we can override at deployment time.